gluten-core/src/main/scala/io/glutenproject/execution/FileSourceScanExecTransformer.scala

```diff
@@ -108,8 +108,9 @@ class FileSourceScanExecTransformer(
   override def hashCode(): Int = super.hashCode()
 
   override protected def doValidateInternal(): ValidationResult = {
-    if (hasMetadataColumns) {
-      return ValidationResult.notOk(s"Unsupported metadataColumns scan in native.")
+    if (!unsupportedColumns.isEmpty) {
+      return ValidationResult.notOk(
+        s"Unsupported columns in native scan: ${unsupportedColumns.mkString(",")}")
```

Comment:
@zhli1142015 Can we modify the reason for unsupport to: 'Unsupported columns in native scan due to column name ambiguity in Velox. Please consider renaming the column.'? It is more readable.